packages/manager/src/features/CloudPulse/Widget/CloudPulseWidget.tsx

```diff
@@ -140,6 +140,7 @@ export const CloudPulseWidget = (props: CloudPulseWidgetProperties) => {
     widget: widgetProp,
   } = props;
   const flags = useFlags();
+  const scaledWidgetUnit = React.useRef(unit === 'Bytes' ? 'B' : unit);
```

Comment:
```suggestion
  const scaledWidgetUnit = React.useRef(generateCurrentUnit(unit));
```

https://github.com/nikhagra-akamai/manager/blob/0bc9d73c4f351b6718061891cc6a9036ca4dff89/packages/manager/src/features/CloudPulse/Utils/unitConversion.ts#L222